olive.c

```diff
@@ -114,6 +114,7 @@ OLIVECDEF void olivec_blend_color(uint32_t *c1, uint32_t c2)
     r1 = (r1*(255 - a2) + r2*a2)/255; if (r1 > 255) r1 = 255;
     g1 = (g1*(255 - a2) + g2*a2)/255; if (g1 > 255) g1 = 255;
     b1 = (b1*(255 - a2) + b2*a2)/255; if (b1 > 255) b1 = 255;
+    a1 = a1 + a2;                     if (a1 > 255) a1 = 255;
```

Comment:
What's the main idea behind adding alphas together? Does it have a deeper underlying meaning or is it purely a hack to make one specific case of `olivec_fill` working on a completely zeroed out canvas?

Reply:
Yep, it's to make it possible to produce images with semi-transparency. No other meaning.

Reply:
@kolumb got it! Thank you for this pull request! I don't think I want to accept it in it's current form, but it gave me some food for thoughts. Thanks!